### PR TITLE
Add actions/actions.hello.yaml

### PR DESCRIPTION
**Pull request by warrenvw**
Resolve https://github.com/StackStorm/st2-docker/issues/9

### CODE REVIEW

File: examples/actions/actions.hello.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+runner_type: "local-shell-cmd"
+description: "Says hello with a shell command"
+entry_point: ""
+parameters:
+  cmd:
+    immutable: true
+    default: "echo Hello {{name}}!"
+  name:
+    type: string
+    description: "Command line arguments passed to rm"
```
Comment (lakshmi-kannan) [inline examples/actions/actions.hello.yaml]: Copy pasta

File: examples/actions/actions.hello.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    default: "dude"
```
Comment (lakshmi-kannan) [inline examples/actions/actions.hello.yaml]: Be inclusive!

File: examples/actions/actions.hello.yaml (hunk 1 above)
Comment (warrenvw) [inline examples/actions/actions.hello.yaml]: .. with pesto or spaghetti sauce?

File: examples/actions/actions.hello.yaml (hunk 2 above)
Comment (warrenvw) [inline examples/actions/actions.hello.yaml]: That'll teach me to use something provided to me. :)

**Pull request merged**